src/tree/localProject/LocalFunctionsTreeItem.ts

```diff
@@ -34,7 +34,9 @@ export class LocalFunctionsTreeItem extends FunctionsTreeItemBase {
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
-        if (this.parent.isIsolated) {
+        const isParentPythonV2Plus = this.parent.langauge === ProjectLanguage.Python && this.parent.languageModel && this.parent.languageModel > 1;
```

Comment:
```suggestion
        const isParentPythonV2Plus = this.parent.language === ProjectLanguage.Python && this.parent.languageModel && this.parent.languageModel > 1;
```

Reply:
It's remarkable that spelling checks still aren't a common thing in linters.  Updated.